content/composer/how-to-install.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
please remove this page, use the current "installation.md" page instead and add the "change the default types" section to it.